tests/monitoring/monitoring_service/test_handlers.py

```diff
@@ -423,7 +426,7 @@ def test_action_monitoring_triggered_event_handler_does_not_trigger_monitor_call
     assert channel.closing_tx_hash is None
 
 
-def test_action_monitoring_triggered_event_handler_does_trigger_monitor_call(  # noqa
+def test_action_monitoring_triggered_event_handler_with_sufficient_balance_does_trigger_monitor_call(  # noqa
     context: Context,
 ):
     """ Tests that `monitor` is called when the ActionMonitoringTriggeredEvent is triggered
```

Comment:
Please also update the docstrings for this and next test.

Reply:
good point, done